clients/pkg/promtail/targets/docker/target.go

```diff
@@ -109,22 +107,22 @@ func (t *Target) processLoop(ctx context.Context) {
 	}
 
 	// Start transferring
-	rstdout, wstdout := io.Pipe()
-	rstderr, wstderr := io.Pipe()
+	cstdout := make(chan []byte, 1)
+	cstderr := make(chan []byte, 1)
```

Comment:
I thought that there was not much point in making it large. But since each channel element here are mostly small and at most 16 kiB, it would probably not hurt to pick a larger number. It could be e.g. 16 instead?

Reply:
I've now changed this to be unbuffered because this seems to be the common practice in the rest of promtail, and I don't have a particular motivation for doing this differently.